### ISSUE TITLE
why ClientLogger.getLog use jar self log file path 

### ISSUE THREAD

**Issue by itmyhome** [apache/rocketmq#338]
hello , RocketMQ-client version 4.2.0 
ClientLogger.java add code block line 92 - 99
```
else if (classType.getName().equals("org.apache.logging.slf4j.Log4jLoggerFactory")) {
                    Class<?> joranConfigurator = Class.forName("org.apache.logging.log4j.core.config.Configurator");
                    Method initialize = joranConfigurator.getDeclaredMethod("initialize", String.class, String.class);
                    if (null == logConfigFilePath) {
                        initialize.invoke(joranConfigurator, "log4j2", log4J2ResourceFile);
                    } else {
                        initialize.invoke(joranConfigurator, "log4j2", logConfigFilePath);
                    }
                }
```
Due to  Can't load project log4j2.xml, instead of log4j_rocketmq_client.xml
can't print project self log, How to slove it ?
thank you !

**Comment by crazyman2010** [comment]
I have this question too, the log4j2_rocketmq_client.xml covered my log4j2.xml  , and then output:
main WARN No Root logger was configured, creating default ERROR-level Root logger with Console appender

**Comment by lollipopjin** [comment]
RocketMQ has replaced slf4j api with InternalLogger, as you can refer to https://github.com/apache/rocketmq/pull/221.

**Comment by vongosling** [comment]
Yes, This is a known bug, which has been fixed in the 4.3.0, will be released soon.
